frontend/src/components/404/index.tsx

```diff
@@ -16,14 +16,16 @@ export default function NotFoundComponent() {
         style={{ minHeight: '100vh' }}
       >
         <Grid item>
-          <img src={headlampBrokenImage} alt={t('404 broken headlamp')} />
+          <img src={headlampBrokenImage} alt="" />
```

Comment:
We need some alt text here I think.

Reply:
I don't think so, because the image doesn't add any context needed that is not already given by the other textual info.
So AFAIU, the best practice here is to use either `alt=""` or `aria-hidden="true"`.